doc/content/guides/connecting-gateways/ciscoixm/_index.md

```diff
@@ -0,0 +1,348 @@
+---
+title: "Cisco LoRaWAN Gateway"
+description: ""
+weight: 1
+---
+
+The Cisco LoRaWAN Gateway is a LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.cisco.com/c/en/us/products/routers/wireless-gateway-lorawan/). This page guides you to connect it to {{% tts %}}.
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways.
+2. Cisco LoRaWAN Gateway with latest firmware version `2.0.32`.
```

Comment:
How to check?

Reply:
None of the other gateway documentations explains how to check this information.
For consistency shouldn't we leave it like that?